app/views/trainees/personal_details/edit.html.erb

```diff
@@ -9,7 +9,9 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <%= form_with model: @trainee, local: true do |f| %>
+    <%= form_for(@personal_detail, url: trainee_personal_details_path(@trainee), method: :put, local: true) do |f| %>
+      <%= f.govuk_error_summary %>
```

Comment:
This should be just before `<h1>` on line 8 so that the error summary message is above the page heading.

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/64/commits/68d4927537fb7c0922c9b3f71c06c77c29dd653a